app/models/runner.js

```diff
@@ -52,18 +51,15 @@ module.exports.runTest = async (jobConfig) => {
             statsToRecord--;
         });
         ee.on('done', async (report) => {
-            reportPrinter.print('aggregate', report);
-            delete report.latencies;
-            artilleryReport = report;
             let handleFinishedReport = async () => {
                 try {
                     await reporterConnector.postStats(jobConfig, {
                         phase_status: 'done',
-                        data: JSON.stringify(report)
+                        data: JSON.stringify({message: 'Test Finished'})
                     });
-                    resolve(artilleryReport);
+                    resolve(report);
```

Comment:
i think since report={} if i'm not mistaken, better to `resolve()` so it's not misleading

Reply:
done